test/CommentTester.cs

```diff
@@ -100,18 +100,22 @@ FOR INSERT
 ";
 
         private readonly string AutoGenerateComment = String.Format(
```

Comment:
You should use the same variable from the the production class ~`Database.AutoGenerateComment`  and delete this one on the test class.

This could easily diverge, if someone changes the variable on `Database.cs` and then not know to change tests because all the tests will still pass.